frontend/src/templates/Field/types.ts

```diff
@@ -154,6 +158,15 @@ export type AddressCompoundFieldValues = {
   addressSubFields: AddressAttributes
 }
 
+export type SignatureFieldValues = {
+  type: 'draw'
+  value: [number, number, number][][]
+}
+// | {
+//     type: 'text' // TODO: unused, kept as example of extension
```

Comment:
question: why do we keep something that is unused? what does keeping as example of extension mean

Reply:
> does keeping as example of extension mean

It means that when `type: 'text'` is introduced as a signature type as opposed to only `draw`. 

But agree that we can take this out, seems like it causes some confusion.